src/test/java/com/hazelcast/wm/test/TomcatServer.java

```diff
@@ -45,15 +45,17 @@ public void restart() throws Exception {
         start();
     }
 
-    public void buildTomcat(String sourceDir, String serverXml) throws LifecycleException {
+    public void buildTomcat(String sourceDir, String serverXml) throws LifecycleException, ServletException, IOException {
         tomcat = new Tomcat();
-        File baseDir = new File(System.getProperty("java.io.tmpdir"));
         tomcat.setPort(port);
-        tomcat.setBaseDir(baseDir.getAbsolutePath());
-        Context context = tomcat.addContext("/", sourceDir);
-        context.getServletContext().setAttribute(Globals.ALT_DD_ATTR, sourceDir + "/WEB-INF/" + serverXml);
-        ContextConfig contextConfig = new ContextConfig();
```

Comment:
so this is not used?

Reply:
Yes, the tests run just fine without it.